lib/tapioca/compilers/dsl/active_record_typed_store.rb

```diff
@@ -0,0 +1,170 @@
+# typed: true
+# frozen_string_literal: true
+
+require "parlour"
+require "tapioca/core_ext/class"
+
+begin
+  require "activerecord-typedstore"
+  require "active_record"
```

Comment:
Do we actually need this require?

Reply:
It looks like we don't need this require for this file at all. We just need it for the test cases to run, so let's just add it to the `spec` file.